airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.json

```diff
@@ -39,6 +39,11 @@
                 "description": "Long-term Authorized Access Token.",
                 "airbyte_secret": true,
                 "type": "string"
+              },
+              "advertiser_id": {
```

Comment:
No, it isn't a breaking change. If you left the advertiser id empty it will continue retrieving all data.
It only introduce the possibility to filter data.